src/components/atoms/Trustpilot/Stars/Stars.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('<TrustpilotStars />', () => {
   });
 
   describe('Ratings', () => {
-    const { container, rerender } = render(<Stars />);
+    const { container, rerender } = render(<Stars />, { legacyRoot: true });
```

Comment:
This is the only test in the latest react testing library that requires this.

Reply:
how come its only this test, not others

Reply:
I don't know. Maybe it means we're doing something not so hot in the Trustpilot component. Or we've written the tests in a weird way.